client/cypress/e2e/earn.cy.ts

```diff
@@ -1,14 +1,55 @@
-import { visitWithLoader, mockCoinPricesServer, connectWallet } from 'cypress/utils/e2e';
+import { visitWithLoader, mockCoinPricesServer, connectWallet, GLM_USD } from 'cypress/utils/e2e';
 import { moveTime } from 'cypress/utils/moveTime';
 import { ConnectWalletParameters } from 'cypress/utils/types';
 import viewports from 'cypress/utils/viewports';
 import {
   HAS_ONBOARDING_BEEN_CLOSED,
+  IS_CRYPTO_MAIN_VALUE_DISPLAY,
   IS_ONBOARDING_ALWAYS_VISIBLE,
   IS_ONBOARDING_DONE,
 } from 'src/constants/localStorageKeys';
 import { ROOT_ROUTES } from 'src/routes/RootRoutes/routes';
 
+const changeMainValueToFiat = () => {
+  cy.get('[data-test=Navbar__Button--Settings]').click();
+  cy.get('[data-test=SettingsCryptoMainValueBox__InputToggle]').uncheck();
+  cy.get('[data-test=Navbar__Button--Earn]').click();
```

Comment:
Please use `navigateWithCheck` util. This function is shared in all tests. Please move it to utils.